src/python/WMCore/WebTools/WebAPI.py

```diff
@@ -73,7 +73,7 @@ def json(self, *args, **kwargs):
 #            dict = self.runMethod(args[0], kwargs)
 #            return dict
             res = self.runMethod(args[0], kwargs)
-            if  type(res) is str:
+            if  isinstance(res, str):
```

Comment:
Same comment here. What if `res` is unicode?

Reply:
Here the situation is a bit easier. We do not import `from builints import str`, so `unicode` would not be matched in py2 both with `type(res) is str` and `isinstance(res, str)`. This change does not change behaviour

However, `src/python/WMCore/WebTools/WebAPI.py` is planned to be modernized in slice 7 and we will have to investigate how to modernize this properly. So this is a problem for (not so remote) future us!

Reply:
Gotcha!